pyServer/manifests/windows/min-diagnostic

```diff
@@ -0,0 +1,4 @@
+echo,### EXPERIMENTAL FEATURE ###
+echo,### Registry Hives ###
+copy,/Windows/System32/config/SOFTWARE
+copy,/Windows/System32/config/SYSTEM
```

Comment:
This needs to be a superset of ASC-VMHealth.

Reply:
Fixed!